packages/core/core/src/PackagerRunner.js

```diff
@@ -147,15 +147,20 @@ export default class PackagerRunner {
     let packaged = await packager.package({
       bundle,
       bundleGraph: new BundleGraph(bundleGraph, this.options),
-      sourceMapPath: path.basename(bundle.filePath) + '.map',
+      getSourceMapReference: map => {
+        return bundle.isInline ||
+          (bundle.target.sourceMap && bundle.target.sourceMap.inline)
```

Comment:
Nice! Is there a way to test this path?

Reply:
Not entirely sure what you mean? I've added a unit test that tests inline sourceMaps

Reply:
Ow the tests for target are in `packages/core/core/test/TargetResolver.test.js` haven't really modified that since I introduced sourcemaps. Should be covered by the inline test I wrote anyway